### ISSUE TITLE
Up to date load testing

### ISSUE THREAD

**Issue by siladu** [Consensys/web3signer#906]
We haven't done much testing beyond 10K validators, however customers are asking about (or already running in some cases?) 20K or more validators.

Main things to test would be:
1. Key Loading/startup time
2. Large numbers of signing requests (e.g. looking for database contention issues)

**Comment by siladu** [comment]
Note, we do appear to have a performance test which be easy to tweak to cover 2. https://github.com/Consensys/web3signer/blob/be75f366056348f5f7a36be90eb1096e6e97c86c/acceptance-tests/src/gatling/scala/Eth2SignSimulation.scala

**Comment by non-fungible-nelson** [comment]
good candidate for holesky validators?

**Comment by jframe** [comment]
Definitely, using the holesky validators could be a good way of testing this
